manifests/init.pp

```diff
@@ -1,20 +1,49 @@
+# == Class: homebrew
+#
+# This class is able to install and configure homebrew and brew cask on node.
+#
+# === Parameters
+#
+# [*user*]
+#   User owner of the brew files and folders.
+#   Defaults to nothing.
```

Comment:
> Should we then set it to undef and throw error if hasn't been set by the user?
I don't see the benefit of doing this rather than not assigning it a default value in the first place. In my mind, that would just be us using optional values to emulate a mandatory value -- may as well use a mandatory value in the first place.